src/Uno.UI/UI/Xaml/Window.Android.cs

```diff
@@ -288,6 +295,31 @@ private bool IsNavigationBarTranslucent()
 				|| flags.HasFlag(WindowManagerFlags.LayoutNoLimits);
 		}
 		#endregion
+
+		internal IDisposable OpenPopup(Popup popup)
```

Comment:
We cannot multi-target this code like you did in the `Popup.WithPopupRoot.cs` ? (BTW: weird file name ... but why not ^^)